tests/push/test_productid_updates_versions.py

```diff
@@ -94,7 +87,10 @@ def test_updates_product_versions(monkeypatch, tmpdir):
     # make an item targeting two of the repos
     item = PulpProductIdPushItem(
         pushsource_item=ProductIdPushItem(
-            name="test", src=str(productid), dest=["repo2", "repo7"]
+            name="test",
+            src=str(productid),
+            dest=["repo2", "repo7"],
+            products=FakeCert(["a", "d"]),
```

Comment:
Using a `FakeCert` as a value for `products` attribute doesn't seem to make sense.

I think these Fake classes are just obsolete now, shouldn't it be something like this using the class added into pushsource:

```
products=[Product(version="a"), Product(version="d")]
```